src/main/java/edu/byu/cs/autograder/quality/QualityAnalyzer.java

```diff
@@ -47,6 +47,10 @@ public QualityAnalysis runQualityChecks(File stageRepo) throws GradingException
             throw new GradingException("Error running code quality: " + e.getMessage(), e);
         }
 
+        if(!checkstyleFinished(output)) {
+            return new QualityAnalysis(0, "", "Could not complete code quality analysis. Please go see a TA.");
```

Comment:
I'm not too familiar with the code quality output, so you can feel free to disregard this, but if the checkstyle does crash and the crash outputs some useful information, is it possible to output that in the results?